recipes/souporcell/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "souporcell" %}
+{% set version = "2.5" %}
+{% set github = "https://github.com/wheaton5/souporcell" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: {{ github }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 1404b87e44c4452b2f329abb81ce4a8383ed5b7e94f38ff2815c5bed62ec2c7f
+  patches:
+    - patches/0001-constrain-vcf-dep.patch
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('souporcell', max_pin="x.x") }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('rust') }}
+    - cargo-bundle-licenses
+  run:
+    - python
+    - pysam
+    - pystan <3
+    - pyvcf
+    - numpy
+    - pyfaidx
+    - scipy
+    - bcftools >=1.9
+    - bedtools >=2.28
+    - freebayes >=1.3
+    - htslib >=1.9
+    - hisat2
+    - minimap2 >=2.26
+    - samtools >=1.9
+    - vartrix
+
+test:
+  commands:
+    - ${PREFIX}/bin/souporcell/target/release/souporcell -h
+    - ${PREFIX}/bin/troublet/target/release/troublet -h
+    - compile_stan_model.py -h
+    - consensus.py -h
+    - renamer.py -h
+    - retag.py -h
+    - shared_samples.py -h
+    - souporcell_pipeline.py -h
+
+about:
+  home: {{ github }}
+  license: MIT
+  license_family: MIT
+  license_file:
+    - LICENSE
+    - souporcell/souporcell-THIRDPARTY.yml
+    - troublet/troublet-THIRDPARTY.yml
+  summary: "Clustering scRNAseq by genotypes."
+
+extra:
```

Comment:
```suggestion
extra:
  additional-platforms:
    - linux-aarch64
    - osx-arm64
```
?

Reply:
This depends on `vartrix`, which we do not compile. Instead we currently do binary redistribution of the 10X releases. Someone would first need to work on that.

Also, I had issue with even the `osx-64` build and compiling the (Py)Stan model. For now this will only have `linux-64`.